communitiesuk/funding-service-design-post-award-data-store
# SMD-87 Update message content to be more specific and uncouple from S3

This PR
- Updates content for funding spent validation to help users find errors more easily
- Fix bug where wrong cell numbers were shown
- Uncouple funding spent valdation from S3 local stack by brining TF_funding_allocated file into the repo



### How to test
The docker-runner repo should have the funding allocation CSV removed and the script that uploads it to the local S3 should be edited.

This should be QA’d to ensure validation runs on the dev environment when this is change is deployed and the existing Funding Allocation CSV is removed from the S3 bucket.
